### ISSUE TITLE
CLI warning: Possible EventEmitter memory leak detected

### ISSUE THREAD

**Issue by fellyph** [GoogleChromeLabs/ps-analysis-tool#302]
**Describe the bug**
Running the CLI at https://pagseguro.uol.com.br/ sitemap, the terminal has triggered this warning: 

```
(node:40144) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 16 SIGINT listeners added to [process]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
(node:40144) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 16 SIGTERM listeners added to [process]. Use emitter.setMaxListeners() to increase limit
(node:40144) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 16 SIGHUP listeners added to [process]. Use emitter.setMaxListeners() to increase limit
✓ Done technology in urls 13 - 15 
(node:40144) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 16 exit listeners added to [process]. Use emitter.setMaxListeners() to increase limit
````

**To Reproduce**
1. Run the command: `npm run cli -- -s https://pagseguro.uol.com.br/sitemap.xml`

**Expected behavior**
Run the command without warnings

**Screenshots**
<img width="1098" alt="Screenshot 2023-11-27 at 09 11 04" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/330792/a10726ec-5a8b-4ff6-8bac-c89b1589cabc">

**Desktop (please complete the following information):**
 - OS: macOS 14.0 (23A344)
 - Browser 119.0.6045.159 (Official Build) (arm64) 
 - Version develop

**Additional context**

**Comment by ayushnirwal** [comment]
Thanks @fellyph for filing this issue but its a copy #258 and is solved by PR #260 which is in develop branch right now. I will be closing this issue.
